### ISSUE TITLE
Questions  about code in  diffusion_transformer.py (q_pred)

### ISSUE THREAD

**Issue by Ed-ivan** [cientgu/VQ-Diffusion#28]
Thanks for your excellent works and sharing your code ! 
while I have questions about the  code:
  log_probs = torch.cat(
            [
                log_add_exp(log_x_start[:,:-1,:]+log_cumprod_at, log_cumprod_bt),
                log_add_exp(log_x_start[:,-1:,:]+log_1_min_cumprod_ct, log_cumprod_ct)
            ],
            dim=1
        )
why log_add_exp is used , looking forward your reply!

**Comment by Ed-ivan** [comment]
while  I  notice that in  line 212  of diffusion_transformer.py,
log_qt=self.q_pred(log_x_t,t)  ,which is called  by  q_posterior(self,log_x_start,log_x_t,t) . 
I feel a little confused that why is log_x_t is used but not log_x_start 

looking forward your reply!
